tests/validation/tests/v3_api/test_import_cluster.py

```diff
@@ -76,13 +90,14 @@ def test_generate_rke_config():
             AWS_NODE_COUNT, random_test_name(HOST_NAME))
     assert len(aws_nodes) == AWS_NODE_COUNT
     # Create RKE config
-    rkeconfigpath = create_rke_cluster_config(aws_nodes)
+    cluster_filename = random_test_name("cluster")
+    rkeconfigpath = create_rke_cluster_config(aws_nodes, cluster_filename)
     rkeconfig = run_command("cat " + rkeconfigpath)
     print("RKE Config file generated\n")
     print(rkeconfig)
 
 
-def create_rke_cluster_config(aws_nodes):
+def create_rke_cluster_config(aws_nodes, cluster_filename):
```

Comment:
I think this function is used in `test_create_ha.py` as well. Will need to update there to pass in the `cluster_filename` or set a default for it in this function.

Reply:
Oh wait... ignore this one. It uses its own definition of the function.